pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+docstyle_convention = numpy
+junit_family=xunit2
+python_files = test_*.py
+python_functions = test_*
```

Comment:
Why are these needed?

Reply:
I thought those would be nice to have a fixed definition what is used for testing by `pytest`.

Reply:
Isn't this automatically done by pytest? If you run `python -m pytest` in a top level directory of a repo it doesn't attempt to run every Python file.

Reply:
In my standard setup for unit testing in my projects I use `unittest` as it was in the beginning, so I was not sure about the conventions in this configuration. I added them just to be sure ... I think you're familiar with it, should I just drop them?

Reply:
I think this should all be covdred by the default

Reply:
Ok, I removed it in [8ece4bb](https://github.com/scikit-hep/pylhe/pull/29/commits/8ece4bbf7dedba99c0bb4376a3bed19812d957f1)!